### PR TITLE
LPS-187103 | Test Fix | Master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Ticket: [LPS-187103](https://issues.liferay.com/browse/LPS-187103)**

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/KaleoDesigner.macro [hunk 1]
```diff
@@ -285,6 +285,20 @@ definition {
 		KaleoDesigner.viewNodeNotPresent(nodeNodeLabel = ${nodeNodeLabel});
 	}
 
+	macro deleteNodeInKaleoForms {
+		var key_nodeType = ${nodeNodeLabel};
```
Comment (davisaints) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/KaleoDesigner.macro:289]: The tests described in the ticket were failing due to the macro that is used (deleteNode) only in the Worklow that is in the Process Builder. Visualizing the Workflow in Kaleo Forms is totally different.

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/KaleoDesigner.macro [hunk 2]
```diff
@@ -629,6 +643,14 @@ definition {
 		ScrollWebElementIntoView(locator1 = "KaleoDesignerEditWorkflow#VIEW_FORM_NODE");
 	}
 
+	macro selectNodeInKaleoForms {
```
Comment (davisaints) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/macros/KaleoDesigner.macro:646]: The same here

**Comment by davisaints** [comment]
Hey @patriciajeanperez, still these tests didn't pass because they are blocked by [LPS-178002](https://issues.liferay.com/browse/LPS-178002)

**Comment by davisaints** [comment]
Hey @patriciajeanperez, can you forward?

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**